sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_models.py

```diff
@@ -43,12 +42,13 @@ class SettingSelector:
     """
     Selects a set of configuration settings from Azure App Configuration.
 
-    :param key_filter: A filter to select configuration settings based on their keys.
+    :keyword key_filter: A filter to select configuration settings based on their keys.
     :type key_filter: str
-    :param label_filter: A filter to select configuration settings based on their labels. Default is value is '\0'
-    :type label_filter: str
+    :keyword label_filter: A filter to select configuration settings based on their labels. Default is value is EMPTY_LABEL 
```

Comment:
This is not cross referenced. But at the same time, we are not adding `EMPTY_LABEL` to dunder-ALL either. Which begs the question, should we?

Reply:
Yeah I was wondering that - I don't think it really _must_ be, as it's unlikely anyone would import it to use given that it's already the default. On the other hand I don't think it would hurt either, just for transparency.

Reply:
So, yes or no to exporting it? I am fine either way.

Reply:
No. Unless something breaks in the docs generation at which point we can revisit this.